install/000-install-dependency-operators.sh

```diff
@@ -12,7 +12,7 @@ echo -e "${INFO} Create/update operator group in namespace '${NAMESPACE}'"
 ${YQ} '.spec.targetNamespaces[0] = strenv(NAMESPACE)' ${RESOURCE_PATH}/prometheus/console-operators.operatorgroup.yaml | ${KUBE} apply -n ${NAMESPACE} -f -
 
 if ${KUBE} get packagemanifests amq-streams >/dev/null ; then
```

Comment:
probably not a big deal, but it's still `amq-streams` in the command

Reply:
The name of the package manifest (and other k8s resources) hasn't changed yet.